test/util/factories/drive_factory.dart

```diff
@@ -27,6 +27,8 @@ class DriveFactory extends TripFactory<Drive> {
     NullableParameter<Profile>? driver,
     List<Ride>? rides,
     List<Chat>? chats,
+    // Convenience parameter to set the duration of the trip
+    Duration? duration,
```

Comment:
Oof this hurts a little. Every other parameter is an actual parameter of the class we are generating or createDependencies, and in comes this exception 😬

Reply:
I mean we can talk about creating a `generateFakefromDuration` method, but come on - then we would have to write down every attribute _again_

Reply:
> Yes but that's why I added the comment. I was not 100% sure, but it might be very useful for readability of the test code.
> 
> (Also, not entirely true: See `createDependencies`)

Haha I mentioned `createDependencies`

Reply:
Oh I can't read obviously... Probably only because you did not write it in code style :|

Reply:
> I mean we can talk about creating a `generateFakefromDuration` method, but come on - then we would have to write down every attribute _again_

That would also hurt a little. I don't know what you'd expect here if you were an experienced test writer...

Reply:
i am not sure why we need to set the duration at the moment, but i guess it's okay like this with the comment

Reply:
I'll remove it, and we can add it again when we think of a good usecase